### PR TITLE
Add Abhishek singh information

### PR DESCRIPTION
**Pull request by 5-Abhi-5**
Add details in data.json file

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -89,6 +89,23 @@
       "github":"https://github.com/Chirag-Mathur",
       "linkedin": "https://www.linkedin.com/in/imchiragmathur/",
       "quote": "Give It All !"
+    },
+    {
+          {
+            "name": {"Abhishek Singh"
+            },
```
Comment (nlok5923) [inline data.json:96]: @5-Abhi-5 you need to remove this "}" brances around your info

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks for PR PTAL !!

File: data.json [hunk 2]
```diff
@@ -89,6 +89,23 @@
       "github":"https://github.com/Chirag-Mathur",
       "linkedin": "https://www.linkedin.com/in/imchiragmathur/",
       "quote": "Give It All !"
-    }
+    },
+    {
+
```
Comment (nlok5923) [inline data.json:94]: @5-Abhi-5 please remove the extra lines

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL !!

**Comment by 5-Abhi-5** [comment]
Please review the changes

**Comment by 5-Abhi-5** [comment]
how to resolve merge conflicts

**Review by nlok5923 — APPROVED** [review]

**Pull request merged**